src/confluent_kafka/src/confluent_kafka.h

```diff
@@ -42,28 +42,28 @@
  *  0xMMmmRRPP
  *  MM=major, mm=minor, RR=revision, PP=patchlevel (not used)
  */
-#define CFL_VERSION     0x01080300
-#define CFL_VERSION_STR "1.8.3"
+#define CFL_VERSION     0x010a0000
+#define CFL_VERSION_STR "1.10.0"
 
 /**
  * Minimum required librdkafka version. This is checked both during
  * build-time (just below) and runtime (see confluent_kafka.c).
  * Make sure to keep the MIN_RD_KAFKA_VERSION, MIN_VER_ERRSTR and #error
  * defines and strings in sync.
  */
-#define MIN_RD_KAFKA_VERSION 0x01060000
+#define MIN_RD_KAFKA_VERSION 0x010a00ff
```

Comment:
It requires librdkafka 1.9.0 where ACLs were added, right?
(the current change will break builds)